conftest.py

```diff
@@ -79,6 +79,12 @@ def skipif(items, whole_module=False):
            isinstance(configuration['platform'], Cpu64):
             skipit = "`icx+cpu64` won't work with this test"
             break
+        # Skip if icx or advisor are not available
+        if i not in ('noadvisor',) or \
```

Comment:
shouldn't this rather be

```
if i == 'noadvisor' and (not isinstance(configuration['compiler'], IntelCompiler) or not get_advisor_path())
```

Reply:
hm.... you are right.. I mistakenly saw in the docker image the mpi test being executed and thought it was this...(compared to a local run in which the mpi test was skipped....)

VERY good catch may thanks